### PR TITLE
CSS for adaptive cards

### PR DESCRIPTION
**Pull request by mokandre**
**Description of changes made (Required)**
- Added SCSS file for the adaptive card
- Added condition for not to display fallback text when the adaptive card is rendered

**Associated issue number/tracking link (Required)**
[94104](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-94104)

**The following criteria should be met (Required)**
- [X] [commit guidelines](https://sqbu-github.cisco.com/WebExSquared/web-client/blob/master/CONTRIBUTING.md#github-commit-guidelines) were followed
- [X] Tests have been added for new functionality and all existing tests pass.
- [X] [pushing guidelines](https://sqbu-github.cisco.com/WebExSquared/web-client/blob/master/CONTRIBUTING.md#pushing-to-the-pipeline) are followed

**Reviewers (Required)**
  @adamweeks @lalli-flores

### CODE REVIEW

File: packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    }
+    button {
+      white-space: nowrap;
+      overflow: hidden;
+    }
+  }
+  .ac-image {
+    max-height: none !important;
+  }
+}
```
Comment (lalli-flores) [inline packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss]: New line at the end of file

File: packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss [hunk 2]
```diff
@@ -0,0 +1,271 @@
+@import '@momentum-ui/core/scss/momentum-ui';
+@import "@ciscospark/react-component-activity-text/src/styles";
+
+.activity-item--adaptive-card {
```
Comment (lalli-flores) [inline packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss]: Is there a way to tests this styles? it's hard to know why certain values or properties are changed

File: packages/node_modules/@ciscospark/react-component-activity-post/src/index.js [hunk 3]
```diff
@@ -2,29 +2,40 @@ import React from 'react';
...
 
 function ActivityPost(props) {
   const {
     content,
     renderedComponent,
-    displayName
+    displayName,
+    renderAdaptiveCard,
+    cards,
+    sparkInstance
```
Comment (lalli-flores) [inline packages/node_modules/@ciscospark/react-component-activity-post/src/index.js]: Please rename to `sdkInstance`

File: packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss (hunk 2 above)
Comment (mokandre) [reply packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss]: scss is for the elements/properties of adaptive card .Snapshot contains styles for those properties  in a newly added test

File: packages/node_modules/@ciscospark/react-component-activity-post/src/index.js (hunk 3 above)
Comment (mokandre) [reply packages/node_modules/@ciscospark/react-component-activity-post/src/index.js]: Changed in parent components as well

**Pull request merged**